packages/seeder/src/drive.js

```diff
@@ -33,8 +33,9 @@ class Drive extends EventEmitter {
     this._onUpload = this._onUpload.bind(this)
     this._onUpdate = this._onUpdate.bind(this)
     this._onReady = this._onReady.bind(this)
-
     this._hyperdrive.on('update', this._onUpdate)
+
+    this.getContentAsync = promisify(this._hyperdrive.getContent)
```

Comment:
Unless we want to make `getContentAsync` "visible" from outside (public member), please prefix it with `_`.
```suggestion
    this._getContentAsync = promisify(this._hyperdrive.getContent)
```